app/src/main/java/com/example/campusguide/map/GoogleMapAdapter.kt

```diff
@@ -37,4 +37,18 @@ class GoogleMapAdapter : Map {
     override fun addPolyline(polyOptions: PolylineOptions?): Polyline? {
         return adapted.addPolyline(polyOptions)
     }
+
+    override fun setOnMapClickListener(mapClickListener: MapClickListener) {
+        return adapted.setOnMapClickListener(mapClickListener)
```

Comment:
why return here and the next 2 functions no return?

Reply:
That is a mistake on my part, good catch :)